player/playloop.c

```diff
@@ -803,6 +803,20 @@ int get_cache_buffering_percentage(struct MPContext *mpctx)
     return mpctx->demuxer ? mpctx->cache_buffer : -1;
 }
 
+static void handle_update_subtitles(struct MPContext *mpctx)
+{
+    if (mpctx->video_status == STATUS_EOF)
+        update_subtitles(mpctx, mpctx->playback_pts);
```

Comment:
should return probably, otherwise you might be doing it twice?

Reply:
Done.